### PR TITLE
neofs-adm: change morph set config error message

### LINKED ISSUE #2556: Improve neofs-adm morph set-config error message
## Is your feature request related to a problem? Please describe.
If we set a 'well-known' attribute using invalid type, we currently get the following error:

```
COMMAND: neofs-adm --config /home/runner/work/neofs-node/neofs-node/neofs-testcases/venv.local-pytest/../../neofs-dev-env/neofs-adm.yml morph set-config --rpc-endpoint 'http://morph-chain.neofs.devenv:30333' --alphabet-wallets '../neofs-dev-env/services/ir' HomomorphicHashingDisabled=0.2
RETCODE: 1

STDOUT:
Error: could not parse HomomorphicHashingDisabled's value '0.2' as bool: strconv.ParseBool: parsing "0.2": invalid syntax
```

## Describe the solution you'd like
The error message shows some implementation details. It is better to display something like this:

Error: invalid type for HomomorphicHashingDisabled key, expected bool, got '0.2'

## Don't forget to add labels!
- component label (`neofs-adm`, `neofs-storage`, ...)
- issue type (`enhancement`, `refactor`, ...)
- `goodfirstissue`, `helpwanted` if needed
- does this issue belong to an epic?
- priority (`P0`-`P4`) if already triaged
- quarter label (`202XQY`) if possible

### PR DESCRIPTION
**Pull request by AliceInHunterland**
Changed error messages to hide details about implementation.

Closes #2556.

### CODE REVIEW

File: cmd/neofs-adm/internal/modules/morph/config.go [hunk 1]
```diff
@@ -157,21 +157,21 @@ func parseConfigPair(kvStr string, force bool) (key string, val interface{}, err
 		netmapMaxObjectSizeKey, netmapWithdrawFeeKey:
 		val, err = strconv.ParseInt(valRaw, 10, 64)
 		if err != nil {
-			err = fmt.Errorf("could not parse %s's value '%s' as int: %w", key, valRaw, err)
+			err = fmt.Errorf("invalid type for %s key, expected int, got '%s'", key, valRaw)
```
Comment (roman-khimov) [inline cmd/neofs-adm/internal/modules/morph/config.go:160]: `invalid value...`?

**Review by roman-khimov — COMMENTED** [review]
Please add a CHANGELOG entry.

**Review by carpawell — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -64,6 +64,7 @@ simple Peapod.
 - LOCK objects and regular objects expiration conflicts (#2392)
 - SN responds with a different node information compared to a bootstrapping contract call's argument (#2568)
 - `neofs-cli object put` command processes multiple `--attributes` flags (#2427)
+- `neofs-adm morph set-config` command error messages (#2556)
```
Comment (roman-khimov) [inline CHANGELOG.md:67]: Rebase please, 0.38.0 is out.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Changelog for NeoFS Node
 ## [Unreleased]
 
 ### Fixed
-
```
Comment (roman-khimov) [inline CHANGELOG.md:7]: Some space _after_ the list would be useful. It's hard to read otherwise.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**